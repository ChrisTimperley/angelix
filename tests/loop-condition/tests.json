{
    "1": {
        "executable": "test",
        "arguments": ["2"]
    },
    "2": {
        "executable": "test",
        "arguments": ["3"]
    },
    "3": {
        "executable": "test",
        "arguments": ["4"]
    }
}
