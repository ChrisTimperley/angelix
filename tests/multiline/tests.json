{
    "0": {
        "executable": "test"
    },
    "1": {
        "executable": "test"
    },
    "2": {
        "executable": "test"
    },
    "3": {
        "executable": "test"
    },
    "4": {
        "executable": "test"
    }

}
