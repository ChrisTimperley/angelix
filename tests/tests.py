import unittest

class TestAngelix(unittest.TestCase):

  def test_1(self):
      self.assertEqual(1, 1)

  def test_2(self):
      self.assertTrue(True)
      self.assertFalse(False)

  def test_3(self):
      self.assertEqual(1, 2)

if __name__ == '__main__':
    unittest.main()
