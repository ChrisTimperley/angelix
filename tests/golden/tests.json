{
    "1": {
        "executable": "test",
        "arguments": ["1", "2"]
    },
    "2": {
        "executable": "test",
        "arguments": ["2", "2"]
    },
    "3": {
        "executable": "test",
        "arguments": ["2", "1"]
    }
}
